python_dependency_verifier/python_dependency_verifier.py

```diff
@@ -118,13 +30,17 @@ def string_dependency_to_dict(string_dependency):
     and returns the following dictionary:
     [name, is_locked_to_specific_version, version_locked_to,
     latest version available from pip]"""
-    dependency_dict = re.split("<|>|=", string_dependency)
-    is_locked = False
+    dependency_list = re.split("<|>|=", string_dependency)
+    is_locked = False  # better naming
     if "==" in string_dependency:
         is_locked = True
-    version_relation = string_dependency.replace(dependency_dict[0], "")
-    dependency_dict = [dependency_dict[0], is_locked, version_relation,
-                       get_latest_version_number(dependency_dict[0])]
+    dependency_name = dependency_list[0]
+    version_relation = string_dependency.replace(dependency_name, "")
```

Comment:
I guess this provides you with `==3.1` or something of that sort? Can you please split the two into fields, one `version` and one `version_relation`?